ui/js/controllers/perf/alerts.js

```diff
@@ -37,17 +37,26 @@ perf.factory('PhBugs', [
     }]);
 
 perf.controller(
-    'ModifyAlertSummaryCtrl', ['$scope', '$uibModalInstance', 'alertSummary',
-        function ($scope, $uibModalInstance, alertSummary) {
+    'ModifyAlertSummaryCtrl', ['$scope', '$uibModalInstance', 'alertSummary', 'phAlertSummaryIssueTrackersMap', 'ThLog',
+        function ($scope, $uibModalInstance, alertSummary, phAlertSummaryIssueTrackersMap, ThLog) {
             $scope.title = "Link to bug";
-            $scope.placeholder = "Bug #";
+            $scope.placeholder = "Task #";
```

Comment:
Not really sure why you're changing this from "Bug #"? It's still a bug number correct?